example/spec/parallel/testAPI/collectionSpec.js

```diff
@@ -46,13 +52,45 @@ describe('Collections API', () => {
         bucket: config.bucket,
         key: recordCreatedKey,
       })).toBeResolved();
+      const savedEvent = await getJsonS3Object(config.bucket, recordCreatedKey);
+      const message = JSON.parse(savedEvent.Records[0].Sns.Message);
+      expect(message.event).toEqual('Create');
+      expect(omit(message.record, ['createdAt', 'updatedAt'])).toEqual(omit(collection, ['createdAt', 'updatedAt']));
```

Comment:
Comparing against `collection` is comparing against the record before it was written, whereas what we want to prove is that the written record was sent to the SNS message.

So I think we should do a `getCollection` and use that to compare with the SNS message

Reply:
We also then don't need to omit the timestamps, which makes the comparison more meaningful.

We should do the same thing below, do a `getCollection` after the update and compare against that

Reply:
We also want to compare against the collection from the API here. Again, otherwise . So we want to move this logic into this block:

```
     const originalCollectionFromApi = await getCollection({
        prefix: config.stackName,
        collectionName: originalCollection.name,
        collectionVersion: originalCollection.version,
      });
      expect(originalCollectionFromApi.reportToEms).toEqual(true);
```

Then you can re-use `originalCollectionFromApi` below. We also don't want to omit the timestamps in the comparison

Reply:
We are still omitting properties in the comparison, which is unnecessary if we're comparing against the record fetched from the API